data/coalition/coalition missions.txt

```diff
@@ -3211,3 +3211,653 @@ mission "Heliarchs Buy Jump Drive Mission"
+			`	"That's reassuring... but, how am I even going to get there?" she queries.`
+			choice
+				`	"I'll take you there."`
+				`	"I don't think you can."`
+					goto decline
+			`	"Do I have to pay anything? I can't afford off-world transport at all, let alone to a completely different segment of the galaxy! Actually, I know - I'll give you a copy of Beyond the Hyperdrive! Thank you so much! Just wait for a while. I'll go over to my house and bring what I'll need for the trip, and then we can go to whatever this Coalition thing is," says Anna.`
+			`	As you wait for Anna, you look over your maps of Coalition space to double-check what planet the Coalition book club is on. After a quick look, you find <destination>.`
+			`	After approximately an hour of waiting, Anna appears outside of your ship, knocking on the hatch. You open it to let her inside and prepare to leave the spaceport.`
+				accept
+			label decline
+			`	Filled with disappointment, Anna says, "Well, I'm afraid I have other things to do, then. Thank you for the stories, at least." She leaves.`
+				decline
+			label bookcopy
+			action
+				set "beyond the hyperdrive - asked for book copy"
+			`	"I'm afraid I can't afford to give one away right now, but I'm happy to show it to you. I assume you already know that it's the story of a group of humans who steal the Quarg's jumping technology and see what's truly beyond the hyperlanes we've known for so long."`
+				goto "beyond the hyperdrive description"
+	on accept
+		clear "beyond the hyperdrive - asked for book copy"
+	on decline
+		clear "beyond the hyperdrive - asked for book copy"
+	on enter
+		system
+			government "Coalition" "Heliarch"
+		conversation
+			`As the flames of your jump drive fade away, Anna comes out of her bunk in shock. "What was that? It felt like the whole ship vanished out of our space altogether! Where are we?"`
+			choice
+				`	"My jump drive would like to welcome you to Coalition space."`
+				`	"That was a jump drive."`
+				`	"That's what happens every time I go to Coalition space with a jump drive."`
+			`	"Wow. I can't believe this is happening!" Anna looks out the window, then turns back.`
+			`	"Can I take a look at your jump drive? I'm interested to see how it works," Anna says. You take her to the engine room of <ship>, where she begins taking notes and pictures of the drive.`
+			`	"Well, thank you for bringing me here," Anna says when she finishes. "I didn't sleep too well, so I'll see you later. Goodbye!" She heads back into her cabin, and you settle back into the controls.`
+	npc
+		government "Heliarch"
+		personality heroic opportunistic
+		fleet "Heliarch"
+	on complete
+		set "coalition: struggling author - beyond the hyperdrive"
+		conversation
+			`Anna gives you the book before you both exit the <ship>. The moment you are out, three Kimek approach you and Anna.`
+			`	"Greetings, <first> <last>. This other human you have brought, who she is, may I ask you?"`
```

Comment:
These civilians aren't at least a little shocked to see another human in Coalition space?